connectors/src/resources/zendesk_resources.ts

```diff
@@ -570,6 +583,15 @@ export class ZendeskTicketResource extends BaseResource<ZendeskTicket> {
     };
   }
 
+  getParentInternalIds(connectorId: number): string[] {
+    /// Tickets have two parents: the Tickets and the Brand.
+    return [
+      getTicketInternalId(connectorId, this.ticketId),
```

Comment:
The order is not crucial but important here, we want to be aligned across connectors. Can you double check it is aligned with what we do elsewhere?

Reply:
bottom to top everywhere 👍